.goreleaser.yml

```
@@ -106,6 +106,29 @@ nfpms:
     maintainer: "golangci-lint Team <golangci-releaser@users.noreply.github.com>"
     description: Fast linters Runner for Go
     license: GPLv3
+    section: devel
```

```suggestion
    section: golang
```

Based on Debian documentation, the section must be `golang`:
- https://www.debian.org/doc/debian-policy/ch-archive.html#s-subsections
- https://packages.debian.org/unstable/
- https://packages.debian.org/unstable/golang/

---

Interestingly, `golang` is not listed in the policy manual sections link. But seems appropriate anyway, done in 9c1cf7ef72896e682e58b9775f3851fdb8691fb9